src/handlers/shared/get-user-role.ts

```diff
@@ -0,0 +1,25 @@
+import { Context } from "../../types";
+
+interface MatchingUserProps {
+  role: string;
+  limit: number;
+}
+
+export async function getUserRole(context: Context, user: string): Promise<MatchingUserProps> {
```

Comment:
I've changed the name to `get-user-task-limit-by-role.ts` . I think this is  as expressive as it could get. It specifies exactly what the function does and what it returns